src/test/resources/test.template.properties

```diff
@@ -40,6 +40,12 @@ test.chromedriver.path=
 test.timeout=15
 
 ###############################################################################
+# Set to true to enable GodMode. Please read the GodMode documentation if you are
+# not yet familiar with it, and use with care. Remember to set back to false when
+# done and rerun the test(s).
+test.godmode.isenabled=false
```

Comment:
I think `test.godmode.enabled` is a better name because the corresponding `TEST_PROPERTIES` field is `IS_GOD_MODE_ENABLED`.
Another thing is whether godmode or god mode (you get what I mean), but that's just nitpicking :p